style.css

```diff
@@ -0,0 +1,854 @@
+}
+
+.skill-list {
+  padding: 10% 0;
+}
+
+.skill-list > ul {
+  padding-inline-start: 2%;
+}
+
+.skills {
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  width: 97%;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.skills > span > img:hover {
+  color: var();
+  background-color: var(--light-text-color);
+}
+
+hr.line {
+  border-top: 1px solid #dfe1e6;
+  margin-inline-start: 0;
+  width: 327px;
+}
+
+.add-skills {
+  background: #f7f7f9;
+  width: 303px;
+  height: 66.72px;
+  border-radius: 8px;
+  padding: 2% 5%;
+  margin: 4% 0;
+  display: flex;
+  align-items: center;
+  gap: 15px;
+}
+
+.language-extend {
+  padding-inline-start: 0;
+}
+
+.inside-text {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+}
+
+.content {
+  border-top-left-radius: 90px;
+  text-align: center;
+  color: white;
+  margin-top: -718px;
+  padding-top: 40px;
+  padding-bottom: 20px;
+  background-color: var(--footer-background);
+}
+
+.content > h1 {
+  color: white;
+  margin-top: 2em;
+}
+
+.text-form {
+  padding: 0 10px;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  max-width: 330px;
+  margin: 0 auto;
+  gap: 30px;
+}
+
+input {
+  height: 50px;
+  padding: 13px;
+  font-weight: 400;
+  font-size: 17px;
+  border-radius: 10px;
+  line-height: 20px;
+  color: #172b4d;
+  border: 1px solid#CFD8DC;
+}
+
+textarea {
+  border-radius: 10px;
+  color: black;
+  padding: 10px;
+  border: 1px solid#CFD8DC;
+}
+
+button {
+  width: 140px;
+  height: 48px;
+  border-radius: 10px;
+  background-color: white;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: black;
+  border: 1px solid#CFD8DC;
+}
+
+.submit {
+  transition: width 2s, height 4s;
+}
+
+.eclipse {
+  content: url("./images/contact form shapes-mobile.svg");
+  position: relative;
+  top: 32px;
+  left: 165px;
+}
+
+.round {
+  border-top: 2px solid #0d141e;
+  margin: 0 auto;
+  width: 100px;
+  border-radius: 10px;
+}
+
+/* effects */
+.more-button:hover,
+.check-button:hover {
+  background-color: #6070ff;
+  color: var(--gray--light-color);
+}
+
+.more-button-1:hover {
+  color: var(--black-color);
+  animation: glowing 20s linear infinite;
+  opacity: 0;
+  transition: opacity 9s ease-in-out;
+  border-radius: 10px;
+  background: linear-gradient(45deg, #f00, #ff7300, #fffb00, #48ff00, #00ffd5, #002bff, #7a00ff, #ff00c8, #f00);
+}
+
+button:hover {
+  background: linear-gradient(45deg, #f00, #ff7300, #fffb00, #48ff00, #00ffd5, #002bff, #7a00ff, #ff00c8, #f00);
+  opacity: 1;
+  outline: 0;
+  color: var(--black-color);
+  line-height: 20px;
+  position: relative;
+  text-align: center;
+  letter-spacing: 1px;
+  text-decoration: none;
+  font-weight: 400px;
+  font-family: sans-serif;
+  text-transform: uppercase;
+}
+
+button:disabled {
+  border: 1px solid #c1c7d0;
+  color: #5e6c84;
+}
+
+.social-link:hover {
+  color: #5e6c84;
+}
```

Comment:
- [ ] Kindly make sure, for all buttons you follow the Figma design for the hover effect and others as show below:

your design:
![Screenshot from 2023-01-18 18-31-39](https://user-images.githubusercontent.com/97834160/213249634-a06e2650-1d3e-409f-b0c2-6d1eb46ae08f.png)




Figma:
![Screenshot from 2023-01-18 18-56-02](https://user-images.githubusercontent.com/97834160/213249701-52f03ba1-6711-4214-b219-f561d45ff155.png)